src/Parser.jl

```diff
@@ -3,12 +3,13 @@ module Parser #JSON
 using Compat
 
 #Define ordered dictionary from DataStructures if present
-_HAVE_DATASTRUCTURES = try
-    using DataStructures
-    import DataStructures.OrderedDict
-    true
-catch
-    false
+function __init__()
+    global _HAVE_DATASTRUCTURES = try
```

Comment:
It's only ever used as a boolean afaict - https://github.com/JuliaLang/JSON.jl/blob/0716a1ccc00e63461e03851387390fdcd43d0c35/src/Parser.jl#L322